extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -85,11 +85,35 @@ public function extendImageData( $data, $width ) {
 			'width' => $dimensions[ 'width' ],
 			'thumbnail' => $thumbnail->getUrl(),
 			'key' => urlencode( $data[ 'key' ] ),
-			'media-type' => $data[ 'isVideo' ] ? 'video' : 'image'
+			'media-type' => $data[ 'isVideo' ] ? 'video' : 'image',
+			'mercuryComponentAttrs' => json_encode( [
+				'itemContext' => 'portable-infobox',
+				'ref' => $ref
```

Comment:
Will both refs be needed? This one and https://github.com/Wikia/app/pull/10121/files#diff-023d0e6389c7cd29ccc7a5bc2889afcaR83 ?